src/Client/JClassPatcher.java

```diff
@@ -144,6 +144,7 @@ private void patchGeneric(ClassNode node) {
 			hookClassVariable(methodNode, "client", "qd", "I", "Game/Camera", "fov", "I", false, true);
 
 			hookClassVariable(methodNode, "client", "Fe", "Z", "Game/Client", "show_bank", "Z", true, false);
+			hookClassVariable(methodNode, "client", "Fe", "Z", "Game/Client", "show_combat", "Z", true, false); //this won't work
```

Comment:
Why are we hooking what appears to be the bank interface being drawn as _show_combat_, can _show_bank_ be used instead?